reinforcement_learning/rl_bin_packing_newsvendor_vehicle_routing_ray_customEnv/rl_bin_packing_ray_custom.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>This is technical language, and difficult to understand. Can we add content from the slide deck, including pictures to explain the problem?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>